src/components/StarRating.js

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+
+export default function StarRating({ rating, saveRatingToDrink, id }) {
+  const [hover, setHover] = useState(0);
+
+  return (
+    <RatingContainer>
+      <SubHeading>Rate this drink:</SubHeading>
+      <StarWrapper>
+        {[...Array(5)].map((star, index) => {
+          index += 1;
+          return (
+            <StarRatingButton
+              type="button"
+              key={index}
+              rating={rating}
+              hover={hover}
+              index={index}
+              onClick={() => saveRatingToDrink(index, id)}
+              onMouseEnter={() => setHover(index)}
+              onMouseLeave={() => setHover(rating)}
+            >
+              <span>&#9733;</span>
+            </StarRatingButton>
+          );
+        })}
+      </StarWrapper>
+    </RatingContainer>
+  );
+}
+
+const RatingContainer = styled.div`
+  margin-top: 20px;
+  display: grid;
+  gap: 15px;
+  min-width: 220px;
+  max-width: 70%;
+`;
+
+const SubHeading = styled.p`
```

Comment:
I think a heading element would be more suitable for `SubHeading`than `p`. 🤔

Reply:
Yes I will change this, thank you :)